<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FDEDEF </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="menu_slider">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>210</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_home">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>210</width>
       <height>40</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    padding-left: 16px;
    text-align: left;
	qproperty-iconSize: 20px 20px;
}

QPushButton:hover {
	background-color: red
}

</string>
     </property>
     <property name="text">
      <string>Trang chủ</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/home.png</normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/home.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_logout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>710</y>
       <width>210</width>
       <height>32</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    padding-left: 16px;
    text-align: left;
	qproperty-iconSize: 20px 20px;
}</string>
     </property>
     <property name="text">
      <string>Đăng xuất</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/logout.png</normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/logout.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_create_product">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>210</width>
       <height>40</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    padding-left: 16px;
    text-align: left;
	qproperty-iconSize: 20px 20px;
}

QPushButton:hover {
	background-color: red
}

</string>
     </property>
     <property name="text">
      <string>Thêm sản phẩm</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/home.png</normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/home.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="search" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>916</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
padding: 8px;
border-radius: 16px;</string>
    </property>
    <widget class="QLineEdit" name="input_search">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>900</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>8</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/icons/search-interface-symbol.png</normaloff>../resources/icons/search-interface-symbol.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/ThaoNguyen1/.designer/resources/icons/colored-logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/hambuger.png</normaloff>C:/Users/ThaoNguyen1/.designer/resources/icons/hambuger.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="current_user">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: right</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>1121</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 0;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scroll_area_widget_contents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1119</width>
       <height>649</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
